LoRaEngine/LoraKeysManagerFacade/IoTCentralImp/Definitions/DeviceCollectionResult.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade.IoTCentralImp.Definitions
+{
+    using System.Collections.Generic;
+
+    public class DeviceCollectionResult
```

Comment:
This doesn't appear to be used anywhere? If so, let's remove it.

Reply:
it's used by IEnumerable<Device>.